openedx/stanford/cms/djangoapps/contentstore/views/utilities/bulkupdate.py

```diff
@@ -43,9 +44,11 @@ def _utility_bulkupdate_get_handler(course_key_string):
         "course_handler",
         course_key_string
     )
-    bulkupdate_url = reverse_course_url(
```

Comment:
This was the major culprit..
This "helper" looks-up/reverses a URL by first prepending `contentstore.views`
which isn't very helpful to us now that we're trying to move the file elsewhere.
Remove the helper and just calling `reverse` directly should do the trick here.